### PR TITLE
Fix #3617. Add feature zoomTo and highlight to Identify

### LINKED ISSUE #3617: Highlight GetFeatureInfo results on map
### Issues
 - depends on #3616

### Acceptance criteria
- A toggle button should be present in the top toolbar of the featureinfo button, called "highlight features"
- The button visibility should be configurable
- When the highlight features button is activated, the clicked geometry should be highlighted on map, shown together with the marker, on feature info click.
- The geometry should be highlighted only if the following conditions are satisfied (Otherwise only the marker will be on map) : 
    - The server supports GFI JSON format
    -  The layer is a vector layer and so the GFI can return also the geometry
- If more then one layer has feature info response, only the geometries shown also in the GFI panel will be visible (so only one layer at the same time). 

Preview ( [mockup](https://demo.geo-solutions.it/mockups/mapstore2/annotation_styles/#/viewer/openlayers/new) ): 
image

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
This changes add the button highlight feature the identify toolbar (on top of identify panel). When highlight is available, you can zoom to the selected feature by clicking on the button. 

Limitations: 
 - This functionality is limited to openlayers, with remote vector data. 
 - Leaflet / Cesium map will not highlight show the button. Anyway zoom to feature may be available.
 - Local features (imported shape files) do not have this functionality yet. 
## Issues
 - Fix #3617

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Feature

**What is the current behavior?** (You can also link to an open issue here)
this feature is not present

**What is the new behavior?**
You can highlight the features 

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

### CODE REVIEW

File: web/client/actions/mapInfo.js [hunk 1]
```diff
@@ -193,6 +195,20 @@ function featureInfoClick(point, layer) {
     };
 }
 
+function highlightFeature(enabled) {
```
Comment (mbarto) [inline web/client/actions/mapInfo.js]: This enables / disables the highlightFeature functionality, it doesn't highlight a feature, I would change the name of the action and action creator.

File: web/client/actions/mapInfo.js [hunk 2]
```diff
@@ -193,6 +195,20 @@ function featureInfoClick(point, layer) {
...
+    return {
+        type: HIGHLIGHT_FEATURE,
+        enabled
+    };
+}
+
+function changeIndex(index) {
```
Comment (mbarto) [inline web/client/actions/mapInfo.js]: Changes the index of what?

File: web/client/actions/mapInfo.js (hunk 1 above)
Comment (offtherailz) [reply web/client/actions/mapInfo.js]: ok

File: web/client/actions/mapInfo.js (hunk 2 above)
Comment (offtherailz) [reply web/client/actions/mapInfo.js]: Index of the current response shown in the feature info panel in the list of valid responses (sub-group of responses). 
The property in the state is mapInfo.index. Should I document the action and the reducer's parts?

File: web/client/actions/mapInfo.js (hunk 1 above)
Comment (offtherailz) [reply web/client/actions/mapInfo.js]: any suggestion?

**Pull request merged**